presentation.py

```diff
@@ -99,18 +92,26 @@ def add_text(slide, text):
     p.text = text
 
 def apply_layout(slide, layout):
-    if 'centerTitle' in layout:
-        add_center_title(slide, layout['centerTitle'])
-    if 'topTitle' in layout:
-        add_top_title(slide, layout['topTitle'])
-    if 'sideTitle' in layout:
-        add_side_title(slide, layout['sideTitle'])
-    if 'img' in layout:
-        add_img(slide, layout['img'])
-    if 'sideImg' in layout:
-        add_side_img(slide, layout['sideImg'])
-    if 'text' in layout:
-        add_text(slide, layout['text'])
+    if 'stackDirection' in layout and layout['stackDirection'] == 'horizontal':
```

Comment:
Since all the other keys in `layout` represent visible components in the slides, I prefer that kind of structure:

```yaml
  - title: "This would be horizontal"
    subtitle: "horizontal subtitle"
    img: "image.png"
    style:
        stackDirection: "horizontal"
```